languages/elixir/reference/README.md

```diff
@@ -0,0 +1,145 @@
+# Elixir reference
+
+## Concepts
+
+Below are the concepts that have been identified in Elixir for use in Exercism.
+
+### Functional
```

Comment:
Do users see this breakdown of these concepts as "functional"? If so, we should probably remove immutability, recursion and pipelines since they're all found in procedural and OO languages as well.

Reply:
I'm not sure, I think they are tere because thedocument was scaffoled from c#'s.  I'm not sure of how they will exactly be represented on the v3 site.

May need more direction from core group.